packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -18,6 +18,12 @@ export type Pagination = {
 
 export interface ReleaseDataResponse extends Omit<Release, 'actions'> {
   actions: { meta: { count: number } };
+  createdBy: {
+    id: string;
+    firstname: string;
+    lastname: string;
+    username: string;
+  };
```

Comment:
i think this type is incorrect? I think you might want to look at – https://github.com/strapi/strapi/tree/develop/packages/core/admin/shared/contracts/shared.ts#L46

Because this is a core package, you could share the type from the admin package to avoid re-typing.